.circleci/config.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: quay.io/govcms/govcms-ci
+    steps:
+      - checkout
+      - run: composer validate --strict || true
```

Comment:
any reason why it is `||true`?

Reply:
Probably just informative. But I think it has come from another test i copied.